docker-compose.yml

```diff
@@ -16,6 +16,7 @@ services:
       - ./.env
     environment:
       - DB_URI=${DB_URI}    
```

Comment:
Not sure if it should be in db initialization or cocoa service itself?

Reply:
I think it should be the URL itself for the database which follows this pattern postgres://user:password@hostname/dbname
python -m cocoa.main

Reply:
we could add here for the env var also the BOT_PREFIX: cocoapls. For the discord bot_token, I am not sure... Maybe we could use the GitHub secrets?